src/transformers/onnx/features.py

```diff
@@ -260,6 +261,10 @@ class FeaturesManager:
             "token-classification",
             onnx_config_cls=LayoutLMOnnxConfig,
         ),
+        "luke": supported_features_mapping(
+            "default",
```

Comment:
We should include the other heads that LUKE supports:

* `masked-lm`
* `entity-classification`
* `entity-pair-classification`
* `entity-span-classification`

Now, `masked-lm` is easy to add, but for the `entity-*` tasks you'll need to add new entries in the `_TASKS_TO_AUTOMODELS` mapping in the `FeaturesManager` [here](https://github.com/huggingface/transformers/blob/d8269eb4d55e4e43c9b885807a82cc428783a573/src/transformers/onnx/features.py#L94). One somewhat annoying thing about LUKE is that these new mappings are specific to just one model and we lose the nice correspondence with autoclasses.

@michaelbenayoun do you think it's OK to include LUKE-specific classes inside the `FeaturesManager`?

Reply:
I'm not very familiar with LUKE, but if it does not fit what is currently being done, I think it is okay to add those classes in the `FeaturesManager`